### PR TITLE
added  CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by crustyy**
Signed-off-by: crustyy <muskanbansal919@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  - <a href="#commit-signing">Developer Certificate of Origin</a>
+- Meshery Contribution Flow
+  - <a href="#contributing-docs">Meshery Documentation</a>
+  - <a href="#contributing-meshery">Meshery Backend</a>
+    - <a href="#adapter">Writing a Meshery Adapter</a>
+  - <a href="#contributing-ui">Meshery UI</a>
+Relevant coding style guidelines are the Go Code Review Comments and the Formatting and style section of Peter Bourgon's Go: Best Practices for Production Environments.
+# <a name="contributing">General Contribution Flow</a>
+
+In order to contribute to Meshery, please follow the fork-and-pull request workflow described [here](./CONTRIBUTING-gitflow.md).
```
Comment (leecalcote) [inline CONTRIBUTING.md:19]: I wonder if we can make some of this more generic to the Layer5 community overall.

**Comment by leecalcote** [comment]
Tagging @morrme here in case she has comments to offer. Can't assign @morrme as a reviewer (without this user joining one of the [teams](https://github.com/orgs/layer5io/teams/contributors))

File: CONTRIBUTING.md (hunk 1 above)
Comment (crustyy) [reply CONTRIBUTING.md:19]: ya good idea we can work on it

**Comment by Adeola-Adesoba** [comment]
> Tagging @morrme here in case she has comments to offer. Can't assign @morrme as a reviewer (without this user joining one of the [teams](https://github.com/orgs/layer5io/teams/contributors))

Hi @leecalcote that link doesn't work. Just wanted you to know 😄

**Comment by leecalcote** [comment]
Oh, oops. I didn’t realize it’s a private link. It’s a link to the “contributors” team in the layer5io org.

**Comment by Adeola-Adesoba** [comment]
> Oh, oops. I didn’t realize it’s a private link. It’s a link to the “contributors” team in the layer5io org.

Hi @leecalcote how can I join this team?

**Comment by leecalcote** [comment]
@Adeola-Adesoba, Excellent question. An invite to join has just been sent.

**Comment by Adeola-Adesoba** [comment]
> @Adeola-Adesoba, Excellent question. An invite to join has just been sent.

Thank you! Joined now.

**Pull request merged**